src/core/agent/services/multiSigService.ts

```diff
@@ -116,11 +123,27 @@ class MultiSigService extends AgentService {
       threshold,
       delegateAid
     );
-    const multisigId = result.op.name.split(".")[1];
-    //this will be updated once the operation is done
-    let isPending = true;
-    if (result.op.done || threshold === 1) {
-      isPending = false;
+    let op = result.op;
+
+    const multisigId = op.name.split(".")[1];
+    const isPending = !op.done;
+
+    if (isPending) {
+      op = await waitAndGetDoneOp(
```

Comment:
Lets simplify this block for multi-sig - we should not wait for the operation to complete for 2 seconds like a normal identifier. The reason is a multi sig will almost always take longer to complete.